dcos/marathon.py

```diff
@@ -753,7 +774,14 @@ def update_pod(self, pod_id, pod_json, force=False):
         :rtype: None
         """
 
-        return self._update('pods', pod_id, pod_json, force)
+        response = self._update_req('pods', pod_id, pod_json, force)
+        header_name = 'Marathon-Deployment-Id'
+
+        try:
+            return response.headers[header_name]
+        except KeyError:
+            template = 'Error: missing "{}" header from Marathon response'
```

Comment:
Any particular reason why? It's about the same length, and Python encourages [EAFP](https://docs.python.org/3.4/glossary.html#term-eafp).

```python
deployment_id = response.headers.get(header_name)
if deployment_id is None:
    template = 'Error: missing "{}" header from Marathon response'
    raise DCOSException(template.format(header_name))

return deployment_id
```